From b7292aeaa8c81ac0db78fa9fb9bf8c884d4e3a80 Mon Sep 17 00:00:00 2001
From: "Apekshit(Appy) Sharma" <appy@cloudera.com>
Date: Tue, 2 Jun 2015 14:11:06 -0700
Subject: [PATCH 154/226] Dependency already removed from trunk by HBASE-8674.
 Somehow still exists in cdh pom.xml.

Author: Apekshit
Reason: Unwanted slow dependency
Ref: CDH-28296
---
 pom.xml |   11 -----------
 1 file changed, 11 deletions(-)

diff --git a/pom.xml b/pom.xml
index c93b06e..01efda9 100644
--- a/pom.xml
+++ b/pom.xml
@@ -467,17 +467,6 @@
         <enabled>false</enabled>
       </snapshots>
     </repository>
-    <repository>
-      <id>ghelmling.testing</id>
-      <name>Gary Helmling test repo</name>
-      <url>http://people.apache.org/~garyh/mvn/</url>
-      <snapshots>
-        <enabled>true</enabled>
-      </snapshots>
-      <releases>
-        <enabled>true</enabled>
-      </releases>
-    </repository>
   </repositories>
   <build>
     <pluginManagement>
-- 
1.7.9.5

